lib/report_portal/cucumber/parallel_report.rb

```diff
@@ -18,12 +18,15 @@
 
 require 'parallel_tests'
 require 'sys/proctable'
+require 'fileutils'
 
 require_relative 'report'
 
 module ReportPortal
   module Cucumber
     class ParallelReport < Report
+      $folder_creation_tracking_file = ".reportportal/folder_creation_tracking.lck"
```

Comment:
lets store all temporary files in Temp folder.

Reply:
sounds good to me!